OneCommunityGlobal/HighestGoodNetworkApp
# Edwin - Make total valid weekly summaries editable by owner/admin

# Description

<img width="567" alt="Screen Shot 2023-07-04 at 1 53 10 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/42288987/6966f41b-ef79-4707-b1a0-40a2d379b7ae">

## Main changes explained:
- Created a new component that has a clickable up arrow and down arrow to edit the valid weekly summaries
- Made changes to the code to send a PATCH request every time the weeklySummariesCount property is changed with the up/down arrows

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner user
4. go to Reports -> Weekly Summaries Report -> "Total Valid Weekly Summaries"
5. Use the up arrow and down arrow to change the count. Refresh the page to make sure the latest count is saved

## Screenshots or videos of changes:

Admin/Owner Account:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/42288987/3769d2da-92f6-45ef-bcc8-00bd515b7b64
